### ISSUE TITLE
ANR during ExoPlayer build()

### ISSUE THREAD

**Issue by daniel-tapas** [google/ExoPlayer#10104]
- ExoPlayer version number
`2.16.0`
- Android version
11 ~ 12
- Android device
samsung, vivo

I receive report from Firebase crashlytics.
I called `ExoPlayerBuilder#build()` in `Fragment#onViewCreated()`.
How to fix it?
Thank you.

```
main (native): tid=1 systid=16975
#00 pc 0xd7178 libc.so 
#01 pc 0x9c48c libc.so 
#02 pc 0x5bfa0 libaudioclient.so 
#03 pc 0x66b84 libaudioclient.so 
       at android.media.AudioSystem.newAudioSessionId(AudioSystem.java)
       at android.media.AudioManager.generateAudioSessionId(AudioManager.java:2567)
       at com.google.android.exoplayer2.util.Util.generateAudioSessionIdV21(Util.java:1730)
       at com.google.android.exoplayer2.SimpleExoPlayer.<init>(SimpleExoPlayer.java:463)
       at com.google.android.exoplayer2.ExoPlayer$Builder.buildSimpleExoPlayer(ExoPlayer.java:967)
       at com.google.android.exoplayer2.ExoPlayer$Builder.build(ExoPlayer.java:961)
```

**Comment by tonihei** [comment]
If you got an ANR when generating the audio session id, I assume your system was somehow blocked doing other tasks at the time and never managed to generate the id. I agree this shouldn't happen, but I also don't think we can do something about it inside the player library because requesting an audio session id shouldn't take any time at all and the methods on system services are usually meant to be called on the main thread.

**Comment by tonihei** [comment]
This should be fixed in Media3 1.6.0
